src/backend/cuda/platform.cpp

```diff
@@ -469,8 +482,59 @@ SparseHandle sparseHandle() {
     return cusparseHandles[id].get()->get();
 }
 
+/*
+ * Map giving the minimum device driver needed in order to run a given version of CUDA
+ * https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html
+ * Map linking cuda version to the minimum driver versions for both Linux/Mac and Windows
+ */
+static const std::map<float, std::pair<double, double> > CudaToKernelVersionMap = {
+  { 10.0, { 410.48, 411.31 } },
+  { 9.2, { 396.37, 398.26 } },
+  { 9.1, { 390.46, 391.29 } },
+  { 9.0, { 384.81, 385.54 } },
+  { 8.0, { 375.26, 376.51 } },
+  { 7.5, { 352.31, 353.66 } }
```

Comment:
Nit: If doubles are not necessary, lets move to floats to store this info. Reduces the size of this map.

Reply:
If you are going to keep doubles, then add `f` literal at the end of key floats of map so that it doesn't throw error on some compilers. (Windows CUDA job failed due to this error).

Reply:
converted to floats